lua/neogit/popups/revert/actions.lua

```diff
@@ -1,16 +1,17 @@
 local M = {}
 
+local util = require("neogit.lib.util")
 local git = require("neogit.lib.git")
 local client = require("neogit.client")
 local notif = require("neogit.lib.notification")
 local CommitSelectViewBuffer = require("neogit.buffers.commit_select_view")
 
 ---@param popup any
----@return table
+---@return CommitLogEntry[]
 local function get_commits(popup)
   local commits
-  if popup.state.env.commits[1] then
-    commits = popup.state.env.commits
+  if #popup.state.env.commits > 0 then
+    commits = util.reverse(popup.state.env.commits)
```

Comment:
This is re-reversing it, no? Since it's reversed in the popup constructor too: https://github.com/NeogitOrg/neogit/pull/777/files#diff-28d9b9d8a1e27ff798c4edf6f0b4c356128f7f0c30c9d91d2e39c589d3db9769R102

Reply:
Right, sorry for that one :sweat_smile: 

Going through the popups again to make sure there are no more unintended changes

Reply:
No worries ;)